src/pages/IDO/index.tsx

```diff
@@ -0,0 +1,336 @@
+            userDelegatee !== ZERO_ADDRESS &&
+            JSBI.notEqual(JSBI.BigInt(0), pngBalance?.raw) ? (
+            <TYPE.body fontWeight={500} mr="6px">
+              <FormattedCurrencyAmount currencyAmount={pngBalance} /> {t('votePage.votes')}
+            </TYPE.body>
+          ) : (
+            ''
+          )} */}
+        </WrapSmall>
+        {/* I don't think we need this section
+        {!showUnlockVoting && (
+          <RowBetween>
+            <div />
+            {userDelegatee && userDelegatee !== ZERO_ADDRESS ? (
+              <RowFixed>
+                <TYPE.body fontWeight={500} mr="4px">
+                  {t('votePage.delegatedTo')}
+                </TYPE.body>
+                <AddressButton>
+                  <StyledExternalLink
+                    href={getEtherscanLink(ChainId.FUJI, userDelegatee, 'address')}
+                    style={{ margin: '0 4px' }}
+                  >
+                    {userDelegatee === account ? 'Self' : shortenAddress(userDelegatee)}
+                  </StyledExternalLink>
+                  <TextButton onClick={toggleDelegateModal} style={{ marginLeft: '4px' }}>
+                    ({t('votePage.edit')})
+                  </TextButton>
+                </AddressButton>
+              </RowFixed>
+            ) : (
+              ''
+            )}
+          </RowBetween>
+        )}
+        */}
+
+        {/* This is the piece displayed when no IDOs were found 
+        {allProposals?.length === 0 && (
+          <EmptyProposals>
+            <TYPE.body style={{ marginBottom: '8px' }}>{t('IDOPage.noIDOsFound')}</TYPE.body>
+            <TYPE.subHeader>
+              <i>{t('IDOPage.IDOAnnounced')}</i>
+            </TYPE.subHeader>
+          </EmptyProposals>
+        )} */}
+        
+        {/* This is the primary IDO list. Here, it seems pages are built dynamically*/}
+        {/*
+        {allProposals?.map((p: ProposalData, i) => {
+          return (
+            <Proposal as={Link} to={'/vote/' + p.id} key={i}>
+              <ProposalNumber>{p.id}</ProposalNumber>
+              <ProposalTitle>{p.title}</ProposalTitle>
+              <ProposalStatus status={p.status}>{p.status}</ProposalStatus>
+            </Proposal>
+          )
+        })}
+        */}
+        {
+          <Proposal as={Link} to={'IDO/1'} key={1}>
+            <PngIcon>
+              <img width={'50px'} src={boofinanceIcon} alt="logo" />
+            </PngIcon>
+            <ProposalTitle>Boo Finance by PenguinFinance</ProposalTitle>
+            <ProposalStatus status="upcoming">Upcoming</ProposalStatus>
+          </Proposal> 
+        }
+        {
+          <Proposal as={Link} to={'IDO/2'} key={2}>
+            <PngIcon>
+              <img width={'50px'} src={yaygamesIcon} alt="logo" />
+            </PngIcon>
+            <ProposalTitle>YAY Protocol by Avalaunch</ProposalTitle>
+            <ProposalStatus status="upcoming">Upcoming</ProposalStatus>
+          </Proposal> 
```

Comment:
do we need this data dynamic ?

Reply:
I simplified the display of the list but left the PNGIcon piece because i could not style the icon and text correctly. Reused code to display each item and the icon. Please review

Reply:
I also added different wording to the overall page. Please review and let me know your thoughts